src/facade/dragonfly_connection.h

```diff
@@ -85,6 +85,8 @@ class Connection : public util::Connection {
     std::vector<std::vector<uint64_t>> commands;
   };
 
+  struct MigrationRequestMessage {};
+
```

Comment:
Now that we've decided what kind of pokemon this is, lets add a comment and maybe even change the name to StopQeueMessage? or smth like that